.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -34,6 +34,10 @@ if ! which git ; then
 fi
 
 cd src
+
+RELATIVE_DIR_PATH="$(dirname "${BASH_SOURCE:-$0}")"
```

Comment:
Is `-PjavaVersion=${JAVA_VERSION}` needed below, or are we just relying on the default?

Reply:
Previously, this used the default JAVA_HOME - and potentially installs JAVA 11.  The directory check in the javaConfig.bash ensures that the default (/opt/java/jdk11) exists - if it doesn't set JAVA_HOME